estate/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+{
+    'name': "Real Estate",
+    'version': '1.0',
+    'depends': ['base'],
+    'author': "Mohamed Yousef",
+    'category': 'Sales',
+    'description': """
+    A Real Estate App to ....
+    """,
+    # data files always loaded at installation
+    'data': [
+        'security/ir.model.access.csv',
+        'views/estate_property_views.xml',
+         'views/estate_menus.xml',
+    ],
+    'installable': True,
+    'application': True,
+    'auto_install': False,
```

Comment:
Not for later in your team: some (most) keys in the manifest have default values, so no need to state it if it is the case (for example, `auto_install` is `False` by default, so you could remove it from here)